addons/binding/org.openhab.binding.velux/src/main/java/org/openhab/binding/velux/VeluxBindingConstants.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2016-2017 (c) Guenther Schreiner <guenther.schreiner@smile.de>
```

Comment:
It seems that this file has a missing, outdated or otherwise non standard copyright header, can you fix all of them by running 

`mvn license:format` 

Please commit only the ones for your binding.

Reply:
fixed.